openvpn/_mapdata/init.sls

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# vim: ft=sls
+---
+{#- Get the `tplroot` from `tpldir` #}
+{%- set tplroot = tpldir.split('/')[0] %}
+{%- from tplroot ~ "/map.jinja" import map as mapdata with context %}
+
+{%- do salt['log.debug']('### MAP.JINJA DUMP ###\n' ~ mapdata | yaml(False)) %}
+
+{%- set output_file = '/tmp/salt_mapdata_dump.yaml' %}
```

Comment:
I'm not sure why; it's not because `C:\tmp` already exists in GitHub Actions; but this method doesn't work in Vagrant. This is why I went with the  environment variable approach shown [here](https://github.com/saltstack-formulas/openssh-formula/pull/193#issuecomment-709445531).

Reply:
@dafyddj @baby-gnu So I've done a bit of snooping around and I've found that our Vagrant box has a `C:\temp` folder available.  So with that tiny modification, all is working fine:

```rb
  ✔  `map.jinja` YAML dump: should contain the lines
     ✔  # yamllint disable rule:indentation rule:line-length
     # Windows-8.1
     ---
     bin_dir: C:\Program Files\OpenVPN\bin\
     conf_dir: C:\Program Files\OpenVPN\config
     conf_ext: ovpn
     dh_files:
     - '512'
     dsaparam: false
     external_repo_enabled: false
     external_repo_supported: []
     external_repo_version: stable
     group: nobody
     log_user: null
     multi_services: false
     network_manager_pkgs:
     - network-manager-openvpn
     - network-manager-openvpn-gnome
     pkgs:
     - openvpn
     service: OpenVPNService
     service_function: running
     user: nobody
      is expected to eq "# yamllint disable rule:indentation rule:line-length\n# Windows-8.1\n---\nbin_dir: C:\\Program Files...openvpn-gnome\npkgs:\n- openvpn\nservice: OpenVPNService\nservice_function: running\nuser: nobody\n"
```

We'll need a different dump file for the Vagrant image, no matter what.  See the line `# Windows-8.1` above.

Reply:
using the environment variable TMP/TMPDIR seems a good approach to me, less dependent of a specific implementation of the Vagrant box

Reply:
@daks That would be a "proper" solution but please see my comment here: https://github.com/saltstack-formulas/openvpn-formula/pull/132#discussion_r509146216.

Reply:
I'm not sure it introduces any additional complexity but as you want.